linux_os/guide/services/ssh/ssh_server/sshd_set_idle_timeout/rule.yml

```diff
@@ -1,12 +1,12 @@
 documentation_complete: true
 
-title: 'Set SSH Idle Timeout Interval'
+title: 'Set SSH Client Alive Interval'
```

Comment:
@vojtapolasek @ggbecker @matejak @marcusburghardt Would you have any concern regarding change of a Rule title?

Reply:
I'm ok with that, it shouldn't break any existing `API`.

Reply:
I am also ok with that. I am not aware of any negative consequence. It seems more aligned to the rule description.